js/dactylo.js

```diff
@@ -168,6 +197,11 @@ window.addEventListener('DOMContentLoaded', () => {
     }
   };
 
+  // startup
+  Promise.all([fetchNgrams(), fetchWords(), fetchLayout()])
+    .then(setLessonLevel);
```

Comment:
Also brilliant, I should really learn how to use promisses. I feel like I’m missing out on something **big**

Reply:
It was a game changer in the FirefoxOS days. Now I hear we can do the same with async/await but I’m still used to the Promise way.